en/docs/getting-started/overview.md

```diff
@@ -1,99 +1,113 @@
+![API Publisher Component Architecture]({{base_path}}/assets/attachments/103327648/126556771.png)
 
-WSO2 API Manager provides a state-of-the-art web interface called **WSO2 API Publisher** for API development and management. It is a structured GUI designed for API creators to develop, document, scale and version APIs, while also facilitating more API management-related tasks such as publishing API, monetizating APIs, and promoting.
+API development is usually done by someone who understands the technical aspects of the API, interfaces, documentation, versions, etc., while API management is typically carried out by someone who understands the business aspects of the APIs. In most business environments, API development is a responsibility that is distinct from API publication and management.
 
-The diagram above shows the common lifecycle activities of an API developer/manager.
+WSO2 API Manager provides a state-of-the-art web interface called **WSO2 API Publisher** for API development and management. It is a structured GUI designed for API creators to develop, document, scale and version APIs, while also facilitating more API management-related tasks such as publishing API, monetizing APIs, and promoting.
 
-#### Developer Portal
+The diagram above shows the common lifecycle activities of an API developer/manager.
 
-![]({{base_path}}/assets/attachments/103327648/126556772.png)
+#### API Developer Portal
 
-The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to [self register]({{base_path}}/learn/consume-api/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup), discover, evaluate, subscribe to and consume APIs.
+![API Developer Portal Component Architecture]({{base_path}}/assets/attachments/103327648/126556772.png)
 
+The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to [self-register]({{base_path}}/develop/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup), discover, evaluate, subscribe to and consume APIs.
 
 The diagram above shows common API consumer lifecycle activities:
 
 #### API Gateway
+<!-- FIXME: fix the wso2 esb URL -->
+The API Gateway is a runtime, backend component (an [API Proxy](https://docs.wso2.com/display/EI611/Working+with+APIs)) developed using [WSO2 ESB]({{base_path}}/getting-started/about-this-release/). API Gateway secures, protects, manages, and scales API calls. It intercepts API requests, applies policies such as throttling and security using handlers, and manages API statistics. Upon validation of a policy, the Gateway passes web service calls to the actual backend. If the service call is a token request, the Gateway passes it directly to the [Key Manager](#key-manager).
```

Comment:
In here we have referenced to an EI documentation for the `API Proxy` concept. And also the WSO2 ESB is referenced to `About this Release` page. Appreciate your comments on the links to be referred

Reply:
This seems to be more relevant https://docs.wso2.com/display/EI660/Working+with+Proxy+Services+. Also please use the term WSO2 EI instead of WSO2 ESB.